tests/baselines/reference/importAssertion1(module=esnext).errors.txt

```diff
@@ -0,0 +1,60 @@
+tests/cases/conformance/importAssertion/3.ts(2,11): message TS1444: Dynamic import must only have a specifier and an optional assertion as arguments
+tests/cases/conformance/importAssertion/3.ts(3,11): message TS1444: Dynamic import must only have a specifier and an optional assertion as arguments
+tests/cases/conformance/importAssertion/3.ts(4,11): message TS1444: Dynamic import must only have a specifier and an optional assertion as arguments
+tests/cases/conformance/importAssertion/3.ts(5,12): message TS1444: Dynamic import must only have a specifier and an optional assertion as arguments
+tests/cases/conformance/importAssertion/3.ts(7,11): message TS1444: Dynamic import must only have a specifier and an optional assertion as arguments
+tests/cases/conformance/importAssertion/3.ts(8,11): message TS1444: Dynamic import must only have a specifier and an optional assertion as arguments
+tests/cases/conformance/importAssertion/3.ts(9,11): message TS1444: Dynamic import must only have a specifier and an optional assertion as arguments
+tests/cases/conformance/importAssertion/3.ts(10,11): message TS1444: Dynamic import must only have a specifier and an optional assertion as arguments
+
+
+==== tests/cases/conformance/importAssertion/0.ts (0 errors) ====
+    export const a = 1;
+    export const b = 2;
+    
+==== tests/cases/conformance/importAssertion/1.ts (0 errors) ====
+    import './0' assert { type: "json" }
+    import { a, b } from './0' assert { "type": "json" }
+    import * as foo from './0' assert { type: "json" }
+    a;
+    b;
+    foo.a;
+    foo.b;
+    
+==== tests/cases/conformance/importAssertion/2.ts (0 errors) ====
+    import { a, b } from './0' assert {}
+    import { a as c, b as d } from './0' assert { a: "a", b: "b", c: "c" }
+    a;
+    b;
+    c;
+    d;
+    
+==== tests/cases/conformance/importAssertion/3.ts (8 errors) ====
+    const a = import('./0')
+    const b = import('./0', { assert: { type: "json" } })
+              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+!!! message TS1444: Dynamic import must only have a specifier and an optional assertion as arguments

```

Comment:
Why is this an error? It seems like its correct per the diagnostic message and the spec.

Reply:
Seems my fault.